native_client/javascript/package.json.in

```diff
@@ -37,6 +39,10 @@
       "node-wav": "0.0.2"
     },
     "devDependencies": {
+      "node-gyp": "4.x - 5.x",
```

Comment:
We also can't have the same package in `dependencies` and `devDependencies`, so we need to make a choice.